includes/admin/class-wc-rest-payments-disputes-controller.php

```diff
@@ -135,6 +144,18 @@ public function close_dispute( $request ) {
 		return $this->forward_request( 'close_dispute', [ $dispute_id ] );
 	}
 
+	/**
+	 * Initiate disputes export via API.
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 */
+	public function get_disputes_export( $request ) {
+		$user_email = $request->get_param( 'user_email' );
```

Comment:
I don't see where we add this parameter, is this set somewhere? Otherwise, it looks like it will always fallback to the value of the blog option `admin_email` in the server handling.

Reply:
Started sending it in params now, missed it earlier.